src/rails_admin/base.js

```diff
@@ -0,0 +1,21 @@
+import Rails from "@rails/ujs";
+import jQuery from "jquery";
+import moment from "moment";
+import "./vendor/jquery.pjax";
+import "./vendor/jquery_nested_form";
+import "bootstrap-sass";
+import "moment/min/locales.js";
+import "eonasdan-bootstrap-datetimepicker";
+
+import "./filter-box";
+import "./filtering-multiselect";
+import "./filtering-select";
+import "./nested-form-hooks";
+import "./remote-form";
+import "./sidescroll";
+import "./ui";
+import "./widgets";
+
+Rails.start();
+window.$ = window.jQuery = jQuery;
+window.moment = moment;
```

Comment:
I don't feel good about this part too, but they are needed in specs like this.
https://github.com/railsadminteam/rails_admin/blob/5cd807e04b4624bb39065196fd978339fe15f548/spec/integration/widgets/datetimepicker_spec.rb#L21
Do you have any idea to avoid this workaround?